src/verilog/verilog_typecheck_expr.cpp

```diff
@@ -1223,14 +1223,14 @@ exprt verilog_typecheck_exprt::convert_constant(constant_exprt expr)
   // check representation
 
   std::string::size_type pos=rest.find('\'');
-  unsigned bits=0;
+  std::size_t bits = 0;
   bool bits_given=false;
 
   if(pos!=std::string::npos) // size given?
   {
     if(rest[0]!='\'')
     {
-      bits=atoi(rest.c_str());
+      bits = atol(rest.c_str());
```

Comment:
This is probably safe because of the lexer rules, but I would feel better if this used one of the `safe_string2...` functions instead of `atol`.